### ISSUE TITLE
Can't change ShareX personal folder.

### ISSUE THREAD

**Issue by Jakubk15** [ShareX/ShareX#5781]
I wanted to change personal folder of ShareX to C:\Program Files\ShareX, so I did it in Application Settings, applied changes, restarted ShareX, and deleted the ShareX folder in Documents (previous location). After another ShareX restart, ShareX will reset its all settings, and creating a ShareX folder in Documents.
[logs](https://github.com/ShareX/ShareX/files/7040264/cI4q8GBmhK.txt)

**Comment by p-bojkowski** [comment]
Same here on:

Edition	Windows 11 Enterprise
Version	21H2
OS build	22000.160
Experience	Windows Feature Experience Pack 1000.22000.160.0

**Comment by tasty0tomato** [comment]
Same issue.

**Comment by ourichermath** [comment]
Same thing here.  Using the Microsoft Store app version.  I tried putting it into %APPDATA%\ShareX (which would have put it under c:\user\[username]\appdata\roaming\sharex), but 1) it didn't create the folder, 2) when I moved the folder from Documents to there, it ignored it and re-created the Documents one and 3) after doing that, it blanked out the Personal Folder I set and returned the entry to blank (i.e., the default location in Documents).
